client/src/components/Rating.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
+
+function Rating({ rating, onUpdateRating }) {
+  const [vote, setVote] = useState(rating);
+
+  const handleClickUpVote = () => {
+    const newRating = vote + 1;
+    setVote(newRating);
+    onUpdateRating(newRating);
+  };
+
+  const handleClickDownVote = () => {
+    let newRating = 0;
+    vote > 0 ? (newRating = vote - 1) : (newRating = 0);
+    setVote(newRating);
+    onUpdateRating(newRating);
+  };
+
+  return (
+    <div className="rating">
+      <FaThumbsUp className="icon-2" onClick={handleClickUpVote} />
```

Comment:
It's better to put these icons inside `button` elements. Eg a keyboard user will be able to tab to them.